src/averageAges.js

```diff
@@ -14,12 +14,21 @@
  *
  * @return {number}
  */
+
 function calculateMenAverageAge(people, century) {
-  // write code here
-  // learn how to use array methods like .filter .map .some .every .find .reduce
-  // avoid using loop and forEach
-  // replace `if ()` statement with &&, || or ?:
-  // without nesting
+  const men = people.filter(person => person.sex === 'm');
+  const menInCentry = century
+    ? men.filter(
+      person => Math.ceil(person.died / 100) === century
+    )
+    : men;
```

Comment:
You filter 2 times when there is `century`. Let's implement this using only one filter